jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -144,7 +144,7 @@ async def job_submission_get(job_submission_id: int = Query(...)):
 
     logger.debug(f"Job-submission data: {job_submission_data=}")
 
-    return job_submission_data
+    return JobSubmissionResponse(**job_submission_data)  # type: ignore
```

Comment:
I would prefer to find a better way to do this. We shouldn't be adding ignores for types if we can avoid it.

We can look at explicitly constructing a dictionary from the result rows a la sqlalchemy Row methods: https://docs.sqlalchemy.org/en/14/core/connections.html#sqlalchemy.engine.Row

It would also be possible to set `orm_mode` for the JobSubmissionResponse model and then use the `from_orm()` method to initialize it with the result rows themselves.

Reply:
I've tried `from_orm()`, but it results in an error when we have any data on the column `execution_parameters` (JSONB)